api/base/serializers.py

```diff
@@ -889,6 +889,9 @@ def to_representation(self, value):
                     elif related_type == 'users' and related_class.view_name == 'user_settings':
                         related_id = resolved_url.kwargs['user_id']
                         related_type = 'user-settings'
+                    elif related_type == 'institutions' and related_class.view_name == 'institution-summary-metrics':
+                        related_id = resolved_url.kwargs['institution_id']
+                        related_type = 'institution-summary-metrics'
```

Comment:
I really dislike how we're doing this. Perhaps we should be getting the type from the view serializer instead of assuming the namespace.

Reply:
Yeah, storing it on the view_class would make more sense. We could certainly revisit this some day.